pandas/core/internals/construction.py

```diff
@@ -460,6 +461,8 @@ def treat_as_nested(data) -> bool:
         len(data) > 0
         and is_list_like(data[0])
         and getattr(data[0], "ndim", 1) == 1
+        # TODO(wayd): hack so pyarrow list elements don't expand
+        and not isinstance(data[0], pa.ListScalar)
```

Comment:
I think have is list like return False for pyarrow scalar is less hacky?

Reply:
That's probably true in this particular case, although I'm not sure how it will generalize to all uses of `is_list_like`. Will do more research